package.json

```diff
@@ -87,64 +87,65 @@
     "@patternfly/patternfly": "4.196.7",
     "@patternfly/react-core": "4.221.3",
     "@patternfly/react-table": "4.90.3",
-    "@redhat-cloud-services/frontend-components": "^3.3.15",
+    "@redhat-cloud-services/frontend-components": "^3.9.19",
+    "@redhat-cloud-services/frontend-components-config": "^4.6.22",
     "@redhat-cloud-services/frontend-components-utilities": "^3.2.3",
     "@unleash/proxy-client-react": "^3.1.0",
     "babel-plugin-transform-imports": "^2.0.0",
-    "classnames": "2.3.1",
+    "classnames": "^2.3.1",
     "escape-string-regexp": "^5.0.0",
     "jest-fetch-mock": "^3.0.3",
     "js-cookie": "^3.0.0",
     "load-json-file": "^7.0.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-hook-form": "^7.13.0",
-    "react-query": "^3.21.0",
-    "react-redux": "8.0.2",
+    "react-hook-form": "^7.36.0",
+    "react-query": "^3.39.2",
+    "react-redux": "^8.0.2",
     "react-router-dom": "^5.2.0",
     "redux": "4.2.0",
     "redux-logger": "3.0.6",
-    "redux-promise-middleware": "6.1.2",
-    "ts-loader": "^9.3.0",
-    "typescript": "^4.3.5",
+    "redux-promise-middleware": "^6.1.2",
+    "ts-loader": "^9.4.1",
+    "typescript": "^4.8.4",
     "uuid": "^8.3.2"
   },
   "devDependencies": {
-    "@babel/core": "^7.14.0",
+    "@babel/core": "^7.19.3",
     "@babel/plugin-proposal-object-rest-spread": "7.18.0",
     "@babel/plugin-syntax-dynamic-import": "7.8.3",
     "@babel/plugin-transform-runtime": "7.18.5",
-    "@babel/preset-env": "7.18.2",
+    "@babel/preset-env": "^7.18.2",
     "@babel/preset-flow": "7.17.12",
     "@babel/preset-react": "7.17.12",
     "@babel/preset-typescript": "^7.15.0",
     "@faker-js/faker": "^7.3.0",
-    "@redhat-cloud-services/eslint-config-redhat-cloud-services": "^1.2.1",
-    "@redhat-cloud-services/frontend-components-config": "^4.6.9",
```

Comment:
This package is a hard dependency of our app - I don't think it will run locally without it. I realize this line showed up in my other comment but I was just saying we didn't need to add the package below it at the time